src/event.js

```diff
@@ -5,10 +5,11 @@ define( [
 	"./var/rnothtmlwhite",
 	"./var/slice",
 	"./data/var/dataPriv",
+	"./manipulation/var/rcheckableType",
```

Comment:
Didn't know commits to my branch would automatically show up in the pull request. I thought they'd stay in my fork, and then I could send again to the pull request. I made some changes, but please ignore them. So I'll still need your input to confirm where I'm going with this. :)